content/participants/ttjc@shivambhatia5432/index.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Shivam Bhatia
+type: "Participant"
+name: "Shivam Bhatia"
+github-link: "https://github.com/shivambhatia5432"
+linkedin-link: "https://www.linkedin.com/in/shivam-bhatia-a83ab4185/"
+looking-for: "Internship"
+description: "Full stack developement | Javascript | Nodejs | React | MongoDB/SQL"
+---
+
+Hi! I am Shivam Bhatia, B.Tech CSE third year student at SRMIST, Delhi-NCR. Let me walk you through my introduction, projects and blogs.
+
+
+### introduction
+
+I am Full Stack/MERN stack developer. I love to design my ideas into code. So I am still learning new technologies.
+
+## why are you taking this challenge?
+
+Looking for an organization where I can use the skills I have learnt by devloping my personal projects.
+Want to learn more by working on big projects and want to learn from mentors.
+
+## something interesting about you?
+
+- I love to try something new and creative so that I can use that in my project.
+- On personal basis I like graphic designing and video editing.
+
+### help I need?
+
+I am still learning to develop more and more skills. But sometimes some things are very complex and difficult to understand and with small help I will break it down.
+
+
+### how to contact me?
+
+- [LinkedIn](https://www.linkedin.com/in/shivam-bhatia-a83ab4185/)
+
+### projects
+It is a simple Google Keep clone- https://notes-keeper-app-shivam.netlify.app <br />
+It is a simple weather website that updates background with city images- https://weather-app-shivam.netlify.app <br />
+It is a simple recipe app- https://recipe-app-shivam.netlify.app <br />
+These 3 projects are based on React only and are quite basic because I am new to React but I have many other projects based on HTML/CSS,Javascript,Jquery,MongoDB,SQL,Express,Node.js and EJS that you can see on my Github.
+
```

Comment:
Please add a proper description for each project:
_description_:
_hosted link_:
_github link_: